src/coreclr/vm/arm/stubs.cpp

```diff
@@ -1381,6 +1381,7 @@ VOID StubLinkerCPU::EmitShuffleThunk(ShuffleEntry *pShuffleEntryArray)
 
 void StubLinkerCPU::ThumbEmitTailCallManagedMethod(MethodDesc *pMD)
 {
+    pMD->EnsureSlotFilled();
```

Comment:
This looks like an inoccuous little function. Seems appropriate to add a comment and the missing contracts.